docs/Basic Concepts.md

```diff
@@ -0,0 +1,61 @@
+# Cloudflow - Basic Concepts
```

Comment:
I think we should better avoid spaces in file names as it creates when you link to them via a url.

Reply:
@debasishg I dont see how this document is linked to the rest. Is this pointed from somewhere?

Reply:
@skonto You are correct - I will link this from the Getting Started doc. And I will change the names of the files to NOT have spaces in a separate PR.